lib/integration_helpers/test_case/v4/result_comparer.rb

```diff
@@ -323,15 +323,15 @@ def expected_employment_income_benefits_in_kind
       end
 
       def expected_employment_income_tax
-        expected_gross_income[:employment_income_tax]
+        expected_disposable_income[:employment_income_tax]&.positive? ? -expected_disposable_income[:employment_income_tax] : expected_disposable_income[:employment_income_tax]
```

Comment:
I don't think this is right - the code ALWAYS makes it a negative figure.  But if they are getting a tax (or NI) refund, then it will be a positive figure and show as such I t hink.